frontend/src/services/api.js

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios';
```

Comment:
your code is missing an npm install axios - the code doesn't run for me.

Reply:
I'll reinstall it. I had installed it before, but I'm not sure why it's not working now. It was functioning correctly on my local machine before I pushed the changes.